assets/i18n/ja.json

```diff
@@ -126,8 +126,8 @@
 	"I18N_INTERACTIONS_GRAPH_UPDATE_LABEL": "ラベルを更新",
 	"I18N_INTERACTIONS_GRAPH_UPDATE_WEIGHT": "重みを更新",
 	"I18N_INTERACTIONS_IMAGE_CLICK_SELECT": "[表示する画像を選択]",
-	"I18N_INTERACTIONS_ITEM_SELECTION_NOT_ENOUGH": "{minChoiceNumber, plural, one{少なくとも1個選択してください。} other{少なくとも#個選択してください。}}",
-	"I18N_INTERACTIONS_ITEM_SELECTION_PREVENT_MORE": "{maxAllowableSelectionCount, plural, one{選択できるのは最大で1個です。} other{選択できるのは最大で#個です。}}",
+	"I18N_INTERACTIONS_ITEM_SELECTION_NOT_ENOUGH": "{minChoiceNumber, plural, =1{少なくとも1個選択してください。} other{少なくとも#個選択してください。}}",
```

Comment:
@kevintab95 Should we use `=1` everywhere to make it consistent (And reliable in the future)? Also, is there any way we can enforce this to be the only pattern touse?